config.js

```diff
@@ -71,6 +71,14 @@ const schema = {
       default: 'noreply@staticman.net',
       env: 'EMAIL_FROM'
     }
+  },
+  reCAPTCHA: {
```

Comment:
I know this is the right way to write it, but I'd prefer to keep config properties `camelCased`. Would you mind changing that?